src/DeploymentProvider/Providers/WebAppDeploymentProvider.ts

```diff
@@ -30,20 +31,20 @@ export class WebAppDeploymentProvider extends BaseWebAppDeploymentProvider {
                 let folderPath = await utility.generateTemporaryFolderForDeployment(false, webPackage, PackageType.jar);
                 let output = await utility.archiveFolderForDeployment(false, folderPath);
                 webPackage = output.webDeployPkg;
-                this.deploymentID = await this.kuduServiceUtility.deployUsingZipDeploy(webPackage);
+                this.deploymentID = await this.kuduServiceUtility.deployUsingZipDeploy(webPackage, { slotName: this.actionParams.slotName });

```

Comment:
in case of PackageType.war, deployment success logging is already being taken care of?

Reply:
yes